node/http/server.go

```diff
@@ -9,7 +9,7 @@ import (
 	"time"
 
 	"github.com/NethermindEth/juno/db"
-	"github.com/NethermindEth/juno/grpc"
+	junogrpc "github.com/NethermindEth/juno/grpc"
```

Comment:
The rename is due to go default formatter behavior.

Reply:
I believe this is due to VSCode defaults. Are you able to change the VSCode formatter to `gofumpt` for this project?

Reply:
Can we revert this change? Since it is unrelated.

Reply:
Sure.